File: ALODWebUtility/Common/lodControl.cs
using System;

namespace ALODWebUtility.Common
{
    [Serializable]
    public class lodControl
    {
        private string _field;
        private bool _IsModified;
        private string _name;
        private string _section;
        private string _value;

        public lodControl(string name, string val, string sec, string fld)
        {
            _name = name;
            _value = val;
            _section = sec;
            _field = fld;
        }

        public string Field
        {
            get
            {
                return _field;
            }
            set
            {
                _field = value;
            }
        }

        public bool IsModified
        {
            get
            {
                return _IsModified;
            }
            set
            {
                _IsModified = value;
            }
        }

        public string Name
        {
            get
            {
                return _name;
            }
            set
            {
                _name = value;
            }
        }

        public string Section
        {
            get
            {
                return _section;
            }
            set
            {
                _section = value;
            }
        }

        public string Value
        {
            get
            {
                return _value;
            }
            set
            {
                _value = value;
            }
        }

        public void ToDataRow(DataSets.ControlLodRow row)
        {
            row.name = _name;
            row.section = _section;
            row.field = _field;
            row.value = _value;
        }
    }
}

File: ALODWebUtility/Common/lodControl.vb
Namespace Common

    <Serializable()>
    Public Class lodControl
        Private _field As String
        Private _IsModified As Boolean
        Private _name As String
        Private _section As String
        Private _value As String

        Public Sub New(ByVal name As String, ByVal val As String, ByVal sec As String, ByVal fld As String)
            _name = name
            _value = val
            _section = sec
            _field = fld
        End Sub

        Public Property Field() As String
            Get
                Return _field
            End Get
            Set(ByVal value As String)
                _field = value
            End Set
        End Property

        Public Property IsModified() As Boolean
            Get
                Return _IsModified
            End Get
            Set(ByVal value As Boolean)
                _IsModified = value
            End Set
        End Property

        Public Property Name() As String
            Get
                Return _name
            End Get
            Set(ByVal value As String)
                _name = value
            End Set
        End Property

        Public Property Section() As String
            Get
                Return _section
            End Get
            Set(ByVal value As String)
                _section = value
            End Set
        End Property

        Public Property Value() As String
            Get
                Return _value
            End Get
            Set(ByVal value As String)
                _value = value
            End Set
        End Property

        Public Sub ToDataRow(ByVal row As DataSets.ControlLodRow)

            row.name = _name
            row.section = _section
            row.field = _field
            row.value = _value

        End Sub

    End Class

End Namespace